kubernetes-manifests/accounts-db.yaml

```diff
@@ -32,7 +32,10 @@ apiVersion: v1
 kind: Service
```

Comment:
`kubernetes-manifests/*` is generated by new releases of Bank of Anthos, and as such these changes are going to be erased upon next release. You want those changes to go in `src/` (you'll find the base k8s files in there for each services.

Reply:
Got it. I have dropped the commit with changes to `kubernetes-manifests/*` and added some changes to the `development` overlay for both `accounts-db` and `ledger-db`.